core/src/main/resources/jenkins/model/Messages.properties

```diff
@@ -70,4 +70,4 @@ ParameterizedJobMixIn.build_with_parameters=Build with Parameters
 ParameterizedJobMixIn.build_now=Build Now
 BlockedBecauseOfBuildInProgress.shortDescription=Build #{0} is already in progress{1}
 BlockedBecauseOfBuildInProgress.ETA=\ (ETA:{0})
-BuildDiscarderProperty.displayName=Discard Old Builds
+BuildDiscarderProperty.displayName=Discard old builds
```

Comment:
If it changes this, we should other uppercases examples: `Build now`. :+1: for this change.